public/app/features/variables/pickers/shared/VariableInput.tsx

```diff
@@ -37,7 +39,7 @@ export class VariableInput extends PureComponent<Props> {
         value={value ?? ''}
         onChange={this.onChange}
         onKeyDown={this.onKeyDown}
-        placeholder="Enter variable value"
+        placeholder={t('variable.picker.input', 'Enter variable value')}
```

Comment:
Maybe `variable.picker.input-placeholder` to keep the context?

Reply:
Yes @JoaoSilvaGrafana. I've just done it. It is changed in `VariableOptions` and `VariableList`. I run it locally and it works. Are they ok?